src/utils/MultipartFormDataParser.ts

```diff
@@ -3,10 +3,20 @@ import util from 'util';
 
 import { getEnvOrThrow } from 'app/utils/EnvironmentUtils';
 
+const supportedFileTypes: string[] = getEnvOrThrow('SUPPORTED_MIME_TYPES').split(',');
+
 export const parseFormData = util.promisify(
     multer({
         limits: {
-            fileSize: parseInt(getEnvOrThrow('MAX_FILE_SIZE_BYTES'), 10)
+            fileSize: Number(getEnvOrThrow('MAX_FILE_SIZE_BYTES'))
+        },
+        // @ts-ignore
+        // tslint:disable-next-line: typedef
+        fileFilter (request, file, cb) {
+            if (!supportedFileTypes.includes(file.mimetype)) {
+                return cb(new Error('File not supported'))
```

Comment:
Include file's mime type that is not supported in error message.

Reply:
But this error is caught anyway by the controller so that we can display the message: 
"The selected file must be a TXT, DOC, PDF, JPEG or PNG"

Reply:
For consistency and in line with my other comment advising to use message codes I suggest we throw Multer exception here with custom code and custom message.

Reply:
I understand it error is caught but good practice says that error messages should be meaningful and helpful in troubleshooting. You cannot expect that every single error will be handled, some errors might slip and then good error message is essential in understanding what happened.